libraries/Microsoft.Bot.Builder.Dialogs/WaterfallDialog.cs

```diff
@@ -20,12 +20,35 @@ public class WaterfallDialog : Dialog
         private const string StepIndex = "stepIndex";
         private const string PersistedValues = "values";
 
-        private WaterfallStep[] _steps;
+        private List<WaterfallStep> _steps;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WaterfallDialog"/> class.
+        /// </summary>
+        /// <param name="dialogId">dialog id</param>
```

Comment:
There is room to raise the bar on this particular comment. :)

Reply:
That is what stylecop insists it should be :P''''